_data/simple-icons.json

```diff
@@ -260,6 +260,11 @@
             "hex": "509CF6",
             "source": "https://www.campaignmonitor.com/brand"
         },
+        {
+            "title": "Castorama",
+            "hex": "0078D7",
+            "source": "https://www.castorama.pl/"
```

Comment:
Maybe change this to `https://www.castorama.fr/` as they're a french retailer ([source](https://en.wikipedia.org/wiki/Castorama))